.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -5,66 +5,102 @@ labels: [bug]
+  - id: behavior
+    type: checkboxes
     attributes:
-      label: Puppeteer version
-      description: What version of Puppeteer are you running?
+      label: Bug behavior
+      description: >
+        How does the bug behave? Does it happen very rarely (flaky)? Is there a
+        lack of error (no error)? If there is a PDF problem, make sure the
+        script writes the PDF somewhere in the current working directory. *Note:
+        PDF implies no error.*
+      options:
+        - label: Flaky
+        - label: No error
+        - label: PDF
+  - id: mvce
+    type: textarea
+    attributes:
+      label: Minimal, reproducible example
+      description: >
+        Provide a [minimal, reproducible
+        example](https://stackoverflow.com/help/minimal-reproducible-example).
+        *No need for backticks — this automatically gets formatted into code.*
+      render: TypeScript
     validations:
       required: true
-  - type: input
-    id: node-version
+  - id: error
+    type: textarea
     attributes:
-      label: Node.js version
-      description: What version of Node.js are you running?
+      label: Error string
+      description: >
+        Provide the bug's error. For example, `throw new Error('test')` would
+        have the error `test`. *Do not include the entire error log.*
+      render: Text
     validations:
       required: true
-  - type: input
-    id: npm-version
+  - id: puppeteer-configuration
+    type: textarea
     attributes:
-      label: npm version
-      description: What version of npm are you running?
+      label: Puppeteer configuration
+      description: >
+        Copy and paste your [configuration
+        file](https://pptr.dev/guides/configuration/) (if applicable). *No need
+        for backticks — this automatically gets formatted into code.*
+      render: TypeScript
+  - id: puppeteer-version
+    type: dropdown
+    attributes:
+      label: Puppeteer version
+      description: What supported version of Puppeteer are you running?
+      options:
+        - '19'
+        - '18'
     validations:
       required: true
-  - type: dropdown
-    id: operating-system
+  - id: node-version
+    type: dropdown
     attributes:
-      label: What operating system are you seeing the problem on?
-      multiple: true
+      label: Node version
+      description: What supported version of Node.js are you running?
       options:
-        - Linux
-        - macOS
-        - Windows
+        - '18'
+        - '16'
+        - '14'
     validations:
       required: true
-  - type: textarea
-    id: config
+  - id: npm-version
+    type: dropdown
     attributes:
-      label: Configuration file
-      description: |
-        Please copy and paste your configuration file [[Docs](https://pptr.dev/guides/configuration/)]. 
-        *No need for backticks — this automatically gets formatted into code.*
-      render: JavaScript
-  - type: textarea
-    id: logs
+      label: NPM version
```

Comment:
what if people are using yarn or pnpm?

Reply:
At the moment, we only support NPM. Do we want to support yarn and pnpm?